stable/instana-agent/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create -}}
+kind: ClusterRoleBinding
+apiVersion: {{ template "instana-agent.rbacApiVersion" . }}
```

Comment:
Still configurable. Why?

Reply:
This is not so much configurable, and more of using the same rbac api version in both the `clusterrolebinding.yaml` and `clusterrole.yaml`

Reply:
Right, I see, but I'd say it doesn't make much sense to extract this. Users might find it confusing. You don't do this for other resources, either.

Reply:
Ok. I'll remove it.

Reply:
Removed!